v3/src/components/graph/adornments/adornments-store-utils.ts

```diff
@@ -0,0 +1,154 @@
+  // items only appear in the univariate plot's ruler.
+  // Since the Show Measure Labels option isn't for activating an adornment, but rather for
+  // modifying the behavior of other adornments, it doesn't have an associated registeredAdornment. So we need
+  //  to add it like this.
+  function guaranteeShowMeasureLabels(measureOrGroup: IMeasure, alreadyAdded: boolean) {
+    if (measureOrGroup.type === "Group" && !alreadyAdded) {
+      measureMenuItems.push({
+        checked: theStore.showMeasureLabels,
+        title: "DG.Inspector.showLabels",
+        type: "control",
+        clickHandler: theStore.toggleShowLabels
+      })
+      return true
+    }
+    return alreadyAdded
+  }
+
+  // Add the Connecting Lines checkbox immediately after the Count/Percent checkbox(es). That setting will be used
+  // by the ScatterDots component to determine whether to show connecting lines.
+  function guaranteeAddedConnectingLinesIfScatterplot(thePlotType: PlotTypes, measureOrGroup: IMeasure,
+                                                      alreadyAdded: boolean) {
+    const registeredAdornment = registeredAdornments.find(a => a.type === measureOrGroup.type)
+    if (plotType === "scatterPlot" && registeredAdornment?.type === "Count" && !alreadyAdded) {
+      measureMenuItems.push({
+        checked: theStore.showConnectingLines,
+        title: "DG.Inspector.graphConnectingLine",
+        type: "control",
+        clickHandler: theStore.toggleShowConnectingLines
+      })
+    }
+    return true // The Connecting Lines checkbox was either already added or just added
+  }
+
+  // Add the Intercept Locked option checkbox immediately after the LSRL checkbox. Since the Intercept Locked
+  // option isn't for activating an adornment, but rather for modifying the behavior of other adornments, it
+  // doesn't have an associated registeredAdornment. So we need to add it like this.
+  function guaranteeAddedLockInterceptIfScatterplot(thePlotType: PlotTypes, measureOrGroup: IMeasure,
+                                                      alreadyAdded: boolean) {
+    if (
+      registeredAdornments.find(a => a.type === measureOrGroup.type)?.type === "LSRL" &&
+      plotType === "scatterPlot" && !alreadyAdded
+    ) {
+      const movableLineVisible = theStore.isShowingAdornment(kMovableLineType)
+      const lsrlVisible = theStore.isShowingAdornment(kLSRLType)
+      const disabled = !movableLineVisible && !lsrlVisible
+      measureMenuItems.push({
+        checked: theStore.interceptLocked,
+        disabled,
+        title: "DG.Inspector.graphInterceptLocked",
+        type: "control",
+        clickHandler: theStore.toggleInterceptLocked
+      })
+    }
+    return true // The Connecting Lines checkbox was either already added or just added
+  }
+
+  // The sum of squares of residuals is only shown for scatter plots, and only when the LSRL or Movable Line is
```

Comment:
Is this comment unfinished? Seems like it's maybe at least missing something like "present" at the end?

Reply:
I hate to be so picky but I still find this comment confusing as written. Either there's a word or two missing at the end or it's assumed that it'll be understood that "shown" in the first clause applies to "is" in the second. The latter is not obvious in my opinion. If nobody else feels that way, though, this is the last time I'll mention it.